sparse/_dok.py

```diff
@@ -242,6 +242,55 @@ def nnz(self):
         """
         return len(self.data)
 
+    @property
+    def format(self):
+        """
+        The storage format of this array.
+        
+        Returns
+        -------
+        str
+            The storage format of this array.
+        
+        See Also
+        -------
+        COO.format : Equivalent :obj:`COO` array property.
+        GCXS.format : Equivalent :obj:`GCXS` array property.
+        scipy.sparse.dok_matrix.format : The Scipy equivalent property.
+        
+        Examples
+        -------
+        >>> import sparse
+        >>> s = sparse.random((5,5), density=0.2, format='dok')
+        >>> s.format
+        'dok'
+        """
+        return "dok"
+
+    @property
```

Comment:
This will be highly inaccurate, I suggest you remove it, since it's a Python dict and not a NumPy one.